### PR TITLE
HTML Forms Home Work 6

### PR DESCRIPTION
**Pull request by sdmnc**
image
image

### CODE REVIEW

File: registrationform.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        <input type="text" id="full-name" name="full-name" value=""> <br>
+        <label for="email"> Email:</label>
+        <input type="text" id="email" name="email" value=""> <br>
+        <label for="password"> Password:</label>
+        <input type="password" id="password" name="password" value="">
+        <label for="password"> Confirm Password:</label>
+        <input type="password" id="confirm-password" name="confirm-password" value=""> <br>
+
+        <!-- Gender Input. -->
+        <p>Gender:
```
Comment (IvanCeban) [inline registrationform.html]: Inside `<p>` should be text. Not form elements

Comment (IvanCeban) [reply registrationform.html]: You can put "Gender" into a <label> tag with no "for" attribute. It should work

File: registrationform.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+        <label for="birthday">Birthday:</label>
+        <input type="date" id="birthday" name="birthday">
+
+    </fieldset>
+
+    <fieldset>
+        <legend>Adress Information:</legend>
+        <form action="/action_page.php">
+            <p>Street Address: <textarea name="message" rows="3" cols="20"> </textarea>
```
Comment (IvanCeban) [inline registrationform.html]: Street address, Country and so on are labels. Please use appropriate tags

File: registrationform.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                <label for="method2"> Phone</label>
+                <input type="checkbox" id="method3" name="method3" value="Postal Mail">
+                <label for="method3"> Postal Mail</label><br>
+            </p>
+
+            <p> Favorite Color:
+                <label for="favcolor"></label>
+                <input type="color" id="favcolor" name="favcolor">
+            </p>
+            <p>
```
Comment (IvanCeban) [inline registrationform.html]: what is the reason to wrap a form element into a paragraph?

File: registrationform.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            How did you hear about us?
+            <select>
+                <option value="Internet"> Internet</option>
+                <option value="Radio"> Radio</option>
+                <option value="TV"> TV</option>
+                <option value="Other"> Other</option>
+            </select>
+        </p>
+
+        <p>Feedback/Comments: <textarea name="message" rows="3" cols="20"> </textarea>
```
Comment (IvanCeban) [inline registrationform.html]: missing closing </p> tag

**Review by IvanCeban — CHANGES_REQUESTED** [review]

File: registrationform.html (hunk 1 above)
Comment (sdmnc) [reply registrationform.html]: Ok. I found the problem I think. Thank you !

File: registrationform.html (hunk 3 above)
Comment (sdmnc) [reply registrationform.html]: fixed that too

**Review by IvanCeban — APPROVED** [review]

**Pull request merged**